rust/processor/src/schema.rs

```diff
@@ -238,64 +189,61 @@ diesel::table! {
 
 diesel::table! {
     current_delegated_staking_pool_balances (staking_pool_address) {
-        #[max_length = 66]
         staking_pool_address -> Varchar,
         total_coins -> Numeric,
         total_shares -> Numeric,
         last_transaction_version -> Int8,
         inserted_at -> Timestamp,
         operator_commission_percentage -> Numeric,
-        #[max_length = 66]
         inactive_table_handle -> Varchar,
-        #[max_length = 66]
         active_table_handle -> Varchar,
     }
 }
 
+diesel::table! {
+    current_delegated_voter (delegation_pool_address, delegator_address) {
+        delegation_pool_address -> Varchar,
+        delegator_address -> Varchar,
+        table_handle -> Varchar,
+        voter -> Nullable<Varchar>,
+        pending_voter -> Nullable<Varchar>,
+        last_transaction_version -> Int8,
+        last_transaction_timestamp -> Timestamp,
+    }
+}
+
 diesel::table! {
     current_delegator_balances (delegator_address, pool_address, pool_type, table_handle) {
-        #[max_length = 66]
```

Comment:
was running the command like `DATABASE_URL=postgres://postgres@localhost:5432/postgres diesel migration run ` but #[max_length] annotation's deleted. is it expected? @bowenyang007

Reply:
Need to upgrade your local diesel